packages/kotti-ui/source/next/kotti-field/KtField.vue

```diff
@@ -126,9 +126,11 @@ export default defineComponent({
 			isGroup: boolean
 			getEmptyValue: () => DATA_TYPE
 		},
-		{ emit, refs }: SetupContext,
+		{ emit },
 	) {
+		const helpTextIconRef = ref<Element>(null)
```

Comment:
Does this actually get correctly detected as `Ref<Element | null>` or is this weird again in Vue?

Reply:
no it doesn't, we established that on initialization of ref in Vue, whatever the type is Vue ignores it xD

I think it's not just for the `null` haha